next/components/forms/simple-components/Banner.tsx

```diff
@@ -0,0 +1,38 @@
+import BannerPhone from '@assets/images/banner-phone.png'
+import cx from 'classnames'
+import Image from 'next/image'
+
+import Button from './Button'
+
+type BannerBase = {
+  title: string
+  content: string
+  onPress?: () => void
+  className?: string
+}
+
+const Banner = ({ title, content, onPress, className }: BannerBase) => {
+  return (
+    <div
+      className={cx(
+        'flex-col sm:flex-row flex h-full items-start justify-center p-0 gap-8 rounded-3xl bg-gray-800',
```

Comment:
p-0 again - you shouldn't need that, please remove everywhere where it does not do anything